### ISSUE TITLE
IAServerAdSource NSInvalidArgumentException Crash

### ISSUE THREAD

**Issue by wimbledon** [inner-active/InneractiveAdSDK-iOS#6]
iOS SDK 7.4.0
  - Fyber_Marketplace_SDK (7.4.0):
    - Fyber_Marketplace_SDK/IASDKCore (= 7.4.0)
    - Fyber_Marketplace_SDK/IASDKMRAID (= 7.4.0)
    - Fyber_Marketplace_SDK/IASDKNative (= 7.4.0)
    - Fyber_Marketplace_SDK/IASDKVideo (= 7.4.0)

**Fatal Exception: NSInvalidArgumentException
Invalid domain=nil in -[NSError initWithDomain:code:userInfo:]**

__78-[IAServerAdSource networkingClient:succeededWithHTTPStatusCode:headers:data:]_block_invoke
keyboard_arrow_down

```
Fatal Exception: NSInvalidArgumentException
0  CoreFoundation                 0x1c10563a8 __exceptionPreprocess
1  libobjc.A.dylib                0x1c025bd00 objc_exception_throw
2  CoreFoundation                 0x1c0f5bbe4 -[NSCache init]
3  Foundation                     0x1c19c84a0 -[NSError initWithDomain:code:userInfo:]
4  Foundation                     0x1c19c8404 +[NSError errorWithDomain:code:userInfo:]
5  Musi                           0x102be16dc __78-[IAServerAdSource networkingClient:succeededWithHTTPStatusCode:headers:data:]_block_invoke + 169 (IAServerAdSource.m:169)
6  libdispatch.dylib              0x1c0a64304 _dispatch_call_block_and_release
7  libdispatch.dylib              0x1c0a65884 _dispatch_client_callout
8  libdispatch.dylib              0x1c0a7173c _dispatch_main_queue_callback_4CF
9  CoreFoundation                 0x1c0fe6024 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
10 CoreFoundation                 0x1c0fe0cd4 __CFRunLoopRun
11 CoreFoundation                 0x1c0fe0254 CFRunLoopRunSpecific
12 GraphicsServices               0x1c321fd8c GSEventRunModal
13 UIKitCore                      0x1ee3284c0 UIApplicationMain
14 Musi                           0x10262fb10 main + 14 (main.m:14)
15 libdyld.dylib                  0x1c0a9cfd8 start
```

**Comment by wimbledon** [comment]
Following up here to see if there is a fix...

**Comment by 2energycell** [comment]
@wimbledon thanks for the input, it looks like the issue was fixed in the 7.4.1 version, also there is already the 7.5.0, which just has been released; you are invited to update to the 7.5.0 and to ensure that the issue does not happen anymore; thanks in advance;

**Comment by wimbledon** [comment]
Thank you, @2energycell!
I just need to push the users to update!
